virtualmachine/vsphere/vm.go

```diff
@@ -877,6 +877,16 @@ func (vm *VM) Destroy() (err error) {
 	return nil
 }
 
+//getToolsRunningStatus returns ToolsRunningState as true/false
+func getToolsRunningStatus(status string) bool {
+	ret := false
```

Comment:
Do we need this variable ?
```go
if condition {
     return true
}
return false
```

Reply:
I mostly prefer to have single entry single exit point..Its have its own advantage and disadvantage.

I read about this ....

There are different schools of thought, and it largely comes down to personal preference.

One is that it is less confusing if there is only a single exit point - you have a single path through the method and you know where to look for the exit. On the minus side if you use indentation to represent nesting, your code ends up massively indented to the right, and it becomes very difficult to follow all the nested scopes.

Another is that you can check preconditions and exit early at the start of a method, so that you know in the body of the method that certain conditions are true, without the entire body of the method being indented 5 miles off to the right. This usually minimises the number of scopes you have to worry about, which makes code much easier to follow.